### PR TITLE
Fix to support values with different encodings

### PR DESCRIPTION
**Pull request by cheerfulstoic**
Been running into this and it seems like when the values get quoted they can get turned into different kinds of encodings, which causes an error when they are combined.

Fix included with tests.  Let me know if you have any questions

### CODE REVIEW

**Comment by jkowens** [comment]
@cheerfulstoic I know this is an old PR, but I'm trying to clean up the backlog. Do you think there still a need for this feature? If you could help me understand the purpose that would be great. Also the commit will need to be cleaned up a bit to resolve conflicts and it would be easier to review if the white space changes were eliminated. Thanks!

**Comment by cheerfulstoic** [comment]
It's been a while, but I think it would still be a useful fix.  Github actually allows you to look at diffs without whitespace like so:

https://github.com/zdennis/activerecord-import/pull/85/files?w=1

Feel free to just copy those changes into `master` and see if things still work

File: test/import_test.rb [hunk 1]
```diff
@@ -202,37 +204,49 @@
...
   end
-  
+
   context "with an array of columns and an array of values" do
     it "should import ids when specified" do
       Topic.import [:id, :author_name, :title], [[99, "Bob Jones", "Topic 99"]]
       assert_equal 99, Topic.last.id
     end
+
+    it "should import data with different encodings" do
```
Comment (jkowens) [inline test/import_test.rb]: Should this spec fail without the change to `values_sql_for_columns_and_attributes` above? In my tests it always passes.

Comment (cheerfulstoic) [inline test/import_test.rb]: Interesting... I would say that I think it should fail, but again it's been a while.  Maybe something changed lower down which is doing conversions for us now... ¯\_(ツ)_/¯

**Comment by jkowens** [comment]
I'm going to close this for now, but if anyone runs into similar issues please feel free to reopen.

**Comment by cheerfulstoic** [comment]
:+1:

**Pull request closed without merging**